### PR TITLE
Add wpt commands for interacting with docker image

### PR DESCRIPTION
**Pull request by jgraham**

### CODE REVIEW

File: tools/docker/Dockerfile [hunk 1]
```diff
@@ -14,6 +14,7 @@ RUN apt-get -qqy update \
     fluxbox \
     gdebi \
     git \
+    default-jre \
```
Comment (gsnedders) [inline tools/docker/Dockerfile]: Why are we adding the JRE here?

Comment (jgraham) [reply tools/docker/Dockerfile]: Part of a different patch to support android emulators.

Comment (gsnedders) [reply tools/docker/Dockerfile]: Can you remove it then, if it's a different patch? :)

Comment (jgraham) [reply tools/docker/Dockerfile]: Yes.

File: tools/docker/start.sh [hunk 2]
```diff
@@ -17,13 +17,15 @@ REF=${2:-master}
 
 cd ~
 
-mkdir web-platform-tests
-cd web-platform-tests
+if [ ! -d web-platform-tests ]; then
```
Comment (gsnedders) [inline tools/docker/start.sh]: Maybe I'm being stupid, but is this actually right? Will the Docker image be recreated when REF changes?

File: tools/docker/frontend.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+def build(*args, **kwargs):
+    subprocess.check_call(["docker",
+                           "build",
+                           "--tag", "wpt:local",
+                           here])
+
+
+def parser_run():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--rebuild", action="store_true", help="Force rebild of image")
```
Comment (gsnedders) [inline tools/docker/frontend.py]:
```suggestion
    parser.add_argument("--rebuild", action="store_true", help="Force rebuild of image")
```

File: tools/docker/start.sh (hunk 2 above)
Comment (jgraham) [reply tools/docker/start.sh]: This runs in the docker process and affects the overlay filesystem which has a lifetime which is that of the specific instance, not of the underlying image. So yes, I think this is correct.

**Review by gsnedders — APPROVED** [review]

**Pull request merged**